### DISCUSSION TITLE
How to make a particular tag draggable as false

### DISCUSSION THREAD

**Discussion by srivatsan2607** [GrapesJS/grapesjs#4741] [Q&A]
Hi @artf ,
I hope you are doing well,
I would like to know how to make particular tag elements say a div tag, draggable as false.
As it is not a custom component, I don't know how to set that draggable as false.
Thanks

**Comment by artf** [comment]
You can apply any component property directly via `data-gjs-*` attribute, eg. `<div data-gjs-draggable="false"...`

**Reply by srivatsan2607** [reply]
It worked! Thank you @artf
